cmds/uptool/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/safing/portbase/updater"
+	"github.com/safing/portbase/utils"
+	"github.com/spf13/cobra"
+)
+
+var registry *updater.ResourceRegistry
+
+var rootCmd = &cobra.Command{
+	Use:   "uptool",
+	Short: "A simple tool to assist in the update and release process",
+	Args:  cobra.ExactArgs(1),
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		absPath, err := filepath.Abs(args[0])
+		if err != nil {
+			return err
+		}
+
+		registry = &updater.ResourceRegistry{}
+		return registry.Initialize(utils.NewDirStructure(absPath, 0o755))
```

Comment:
If I understand correctly, these permissions are the same as 0755. I like the explicit octal typing. Maybe we should adopt this everywhere.
It'd be nice if the Go stdlib also did this.